paparazzi-gradle-plugin/src/test/projects/overwrite-on-max-percent-difference/gradle.properties

```diff
@@ -0,0 +1,4 @@
+android.useAndroidX=true
+android.nonTransitiveRClass=false
+
+app.cash.paparazzi.overwriteOnMaxPercentDifference=true
```

Comment:
@geoff-powell with the current name it is not clear if the type of the flag is a `boolean` or a `float`

i.e. someone might (incorrectly) try this: 
`app.cash.paparazzi.overwriteOnMaxPercentDifference=0.01`

If we call it something like `overwriteOnlyOnMaxPercentDifferenceOrGreater` it makes it clearer that it is a boolean.